packages/kbn-grid-layout/grid/grid_panel/resize_handle.tsx

```diff
@@ -42,7 +41,9 @@ export const ResizeHandle = ({
         margin: -2px;
         position: absolute;
         width: ${euiThemeVars.euiSizeL};
+        max-width: 100%;
         height: ${euiThemeVars.euiSizeL};
+        z-index: ${euiThemeVars.euiZLevel9};
```

Comment:
The resize handle was being hidden behind the embeddable renderer, so I had to increase the z-index so that it floats above